cypress/support/fragments/data_import/logs/logsViewAll.js

```diff
@@ -481,4 +483,10 @@ export default {
   noLogResultsFound: () => {
     cy.expect(logsResultPane.find(HTML('No results found. Please check your filters.')).exists());
   },
+
+  collapseButtonClick: () => cy.do(collapseButton.click()),
+
+  expandButtonClick: () => cy.do(expandButton.click()),
+
+  checkSearchPaneCollapsed: () => cy.expect(Pane('Search & filter').absent()),
```

Comment:
```JS 
Pane('Search & filter')
```

is already used in another method, please move it to constant and replace all usages

Reply:
fixed